experiment/bump_e2e_image.sh

```diff
@@ -36,44 +36,40 @@ if [[ -n ${dirty} ]]; then
   exit 1
 fi
 
-TREE="$(dirname ${BASH_SOURCE[0]})/.."
+TREE="$(dirname "${BASH_SOURCE[0]}")/.."
 
 DATE="$(date +v%Y%m%d)"
 TAG="${DATE}-$(git describe --tags --always --dirty)"
-pushd "${TREE}/images/kubekins-e2e"
-make push
-K8S=experimental make push
-K8S=1.11 make push
-K8S=1.10 make push
-K8S=1.9 make push
-K8S=1.8 make push
-popd
+ 
+make -C "${TREE}/images/kubekins-e2e" push
+K8S=experimental make -C "${TREE}/images/kubekins-e2e" push
+K8S=1.11 make -C "${TREE}/images/kubekins-e2e" push
+K8S=1.10 make -C "${TREE}/images/kubekins-e2e" push
+K8S=1.9 make -C "${TREE}/images/kubekins-e2e" push
+K8S=1.8 make -C "${TREE}/images/kubekins-e2e" push
 
 echo "TAG = ${TAG}"
 
 $SED -i "s/\/kubekins-e2e:.*$/\/kubekins-e2e:${TAG}-master/" "${TREE}/images/kubeadm/Dockerfile"
 $SED -i "s/\/kubekins-e2e:v.*$/\/kubekins-e2e:${TAG}-master/" "${TREE}/experiment/generate_tests.py"
 $SED -i "s/\/kubekins-e2e:v.*-\(.*\)$/\/kubekins-e2e:${TAG}-\1/" "${TREE}/experiment/test_config.yaml"
 
-pushd "${TREE}"
 bazel run //experiment:generate_tests -- \
-  --yaml-config-path=experiment/test_config.yaml \
-  --json-config-path=jobs/config.json \
-  --prow-config-path=prow/config.yaml
+  "--yaml-config-path=${TREE}/experiment/test_config.yaml" \
+  "--json-config-path=${TREE}/jobs/config.json" \
+  "--prow-config-path=${TREE}/prow/config.yaml"
 bazel run //jobs:config_sort
-popd
 
 # Scan for kubekins-e2e:v.* as a rudimentary way to avoid
 # replacing :latest.
 $SED -i "s/\/kubekins-e2e:v.*-\(.*\)$/\/kubekins-e2e:${TAG}-\1/" "${TREE}/prow/config.yaml"
+find "${TREE}/config/jobs/" -type f -name \*.yaml -exec $SED -i "s/\/kubekins-e2e:v.*-\(.*\)$/\/kubekins-e2e:${TAG}-\1/" {} \;
```

Comment:
these \ should be \\
https://github.com/kubernetes/test-infra/pull/8032/files#diff-2649426475dffa0e0587adf3e8822a33

Reply:
ok I took #8032 and added that one line :-)